### PR TITLE
[PORT] product_lot_sequence: #1323 from 15.0

### PR DESCRIPTION
**Pull request by vvrossem**
cf. #1323

### CODE REVIEW

**Comment by vvrossem** [comment]
@grindtildeath would you mind reviewing this PR? :pray:

**Review by yankinmax — APPROVED** [review]
Thanks

File: product_lot_sequence/models/stock_move.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+            if not self.display_assign_serial:
+                return super(
+                    StockMove, self.with_context(force_next_serial="")
+                ).action_show_details()
+            # If the sequence was already called once, avoid calling it another time
+            elif self.next_serial:
+                return super(
+                    StockMove, self.with_context(force_next_serial=self.next_serial)
+                ).action_show_details()
+        return super().action_show_details()
```
Comment (SilvioC2C) [inline product_lot_sequence/models/stock_move.py]: A little bit of refactoring:
```suggestion
        if seq_policy in ("product", "global"):
            # If move is not supposed to assign serial pass empty string for next serial
            if not self.display_assign_serial:
                self = self.with_context(force_next_serial="")
            # If the sequence was already called once, avoid calling it another time
            elif self.next_serial:
                self = self.with_context(force_next_serial=self.next_serial)
        return super().action_show_details()
```

**Review by SilvioC2C — APPROVED** [review]
LGTM overall, non-blocking comment

**Review by grindtildeath — APPROVED** [review]

**Comment by grindtildeath** [comment]
ping @OCA/product-maintainers

**Comment by gurneyalex** [comment]
Hello @vvrossem Are you sure you followed the migration process for this PR? I'm surprised not to see any translations commits. 

See the HOWTO steps in https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-16.0

**Comment by paulius-sladkevicius** [comment]
@dreispt looks like a merge stuck.

**Comment by paulius-sladkevicius** [comment]
@dreispt wondering if that will do a trick as https://github.com/OCA/product-attribute/pull/1559/checks?check_run_id=22995777385 is still stuck :-)

**Pull request merged**